store/transfer_processor.go

```diff
@@ -295,29 +307,60 @@ func (s *transferProcessor) process(ctx context.Context, events []*model.Transfe
 			}
 			totalBalances[sub.Group][token] = new(big.Int).Add(tb, d)
 
+			if token != model.ETHAddress {
+				continue
+			}
+
 			// Consider tx fees
-			if f, ok := feeDiff[addr]; ok && token == model.ETHAddress {
+			if f, ok := feeDiff[addr]; ok {
 				if totalFees[sub.Group] == nil {
 					totalFees[sub.Group] = new(big.Int).Set(f)
 				} else {
 					totalFees[sub.Group] = new(big.Int).Add(f, totalFees[sub.Group])
 				}
 			}
+			// Consider miner reward
+			if r, ok := minerRewardDiff[addr]; ok {
+				s.logger.Info("add miner reward to total balance")
+				if totalMinerReward[sub.Group] == nil {
+					totalMinerReward[sub.Group] = new(big.Int).Set(r)
```

Comment:
`totalMinerReward[sub.Group]` should only set, and never add?

Reply:
OK, i remove the `add` function, and return error after event detecting was hit with invalid transfer.  https://github.com/getamis/eth-indexer/pull/54/files#diff-778fd13deb3062e86beaee58b3afff70R177